src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/CanonicalizationDriver.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Security.Cryptography;
+using System.Text;
+using System.Xml;
+using System;
+
+namespace CoreWCF.IdentityModel
+{
+    sealed class CanonicalizationDriver
+    {
```

Comment:
You shouldn't need this class as SignedXml can do the canonicalization via a transform.